community_images/ansible-lint/ironbank/image.yml

```diff
@@ -0,0 +1,33 @@
+name: ansible-lint-ib
+official_name: Ansible-Lint Iron Bank
+official_website: https://github.com/ansible/ansible-lint
+source_image_provider: Platform One
+source_image_repo: registry1.dso.mil/ironbank/opensource/ansible/ansible-lint
+source_image_repo_link: https://registry1.dso.mil/harbor/projects/3/repositories/opensource%2Fansible%2Fansible-lint
+source_image_readme: https://repo1.dso.mil/dsop/opensource/ansible/ansible-lint/-/blob/development/README.md
+rf_docker_link: rapidfort/ansible-lint-ib
+image_workflow_name: ansible_lint_ironbank
+github_location: ansible-lint/ironbank
+report_url: https://us01.rapidfort.com/app/community/imageinfo/registry1.dso.mil%2Fironbank%2Fopensource%2Fansible%2Fansible-lint
+usage_instructions: |
+  $ docker run --rm -it rapidfort:ansible-lint-ib:latest <file/directory_to_be_checked>
+what_is_text: |
+  ansible-lint checks ansible playbooks for practices and behaviour that could potentially be improved.
+disclaimer: |
+  Trademarks: This software listing is packaged by RapidFort. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
+is_locked: "False"
+input_registry:
+  registry: registry1.dso.mil
+  account: ironbank
+repo_sets:
+  - opensource/ansible/ansible-lint:
+      input_base_tag: "24.2."
+      output_repo: ansible-lint-ib
+runtimes:
+  - type: docker
+    script: docker_coverage.sh
+    ansible-lint-ib:
+      entrypoint: sleep
```

Comment:
Why do we need to change the entry point here?

Reply:
The default entrypoint here is `ansible-lint`.
So the way this container works is, it expects subsequent command (flags or path of files/directories to be linted), prints results and exits.

In coverage script, we have to execute multiple commands. Better than re-starting the container again and again for all of commands, it was more appropriate to change the entrypoint to sleep, pass exec_command as infinity. This makes sure container is running, and does not exit. Allows us to exec into it and lint required files using ansible-lint CLI commands.